common/scripted_effects/wc_exodar_effects.txt

```diff
@@ -0,0 +1,376 @@
+﻿create_characters_exodar_effect = {
+	create_character = {
+		template = velen_character_template
+		location = title:c_valaar.title_province
+		name = "Velen"
```

Comment:
First update your branch to master and then add all your spawn character names into
`\localization\english\names` not `replace` folder, updated name folder will appear after merge from master branch